B827EBFFFE6C919A.json

```diff
@@ -3,7 +3,8 @@
     "gateway_ID": "B827EBFFFE6C919A",
     "servers": [
       {
-        "server_address": "router.eu.thethings.network",
+        "gateway_ID": "B827EBFFFE6C919A",
```

Comment:
Do not need to include the gateway ID again inside this section. Please remove and then we merge.